serialization.md

```diff
@@ -472,4 +472,37 @@ The resulting Tar archive for `f60c56784b83` has the following entries:
 /etc/.wh.my-app-config
 ```
 
+In addition to expressing that a single entry should be removed from a lower layer, layers may remove all of the children using and opaque whiteout entry.
+An opaque whiteout entry is a file with the name `.wh..wh..opq` indicating that all siblings are hidden in the lower layer.
+Let's take the following base layer as an example:
+
+```
+etc/
+	my-app-config
+bin/
+	my-app-binary
+	my-app-tools
+```
+
+If we remove all of the children of `bin/` in the next layer, we would have the following:
+
+```
+bin/
+	.wh..wh..opq
+```
+
+We'll call this the _opaque whiteout_ format.
+Using _explicit whiteout_ files, this would be equivalent to the following:
+
+```
+bin/
+	.wh.my-app-binary
+	.wh.my-app-tools
+```
+
+In this case, a unique whiteout file is generated for each entry.
+If there were more children of `bin/` in the base layer, there would be an entry for each.
```

Comment:
what if a child is a directory itself? gonna try that out but would be nice to provide an example if it's not clear instantly (at least to me for now)